calculate_score.py

```diff
@@ -0,0 +1,30 @@
+
+def cal_accuracy_score(given_word: str, 
+                       user_text: str, no_of_backspaces: int) -> float:
```

Comment:
This function returns a Tuple. Figure out what are tuples. Also change the return type of this function to Tuple and then add relevant types in it